index.html

```diff
@@ -17,15 +17,25 @@
     <div class="Main-container">
         <div class="container">
             <header class="header hoverable">
-                <a>My Logo</a>
-                <nav><img src="/photos/Union.png" alt="Hamburger Icon"></nav>
```

Comment:
- [ ] your Images doesn't show up because of case sensitive of your src link it would be better if you try to add a dot before the slash like that `src="./photos/Union.png"` that will make your image show up 
and also do that to all your images link